src/content/guides/author-libraries.md

```diff
@@ -32,11 +32,11 @@ The basic project structure would look like this:
 +    |- ref.json
 ```
 
-Initialize the project with npm, then install `webpack` and `lodash`:
+Initialize the project with npm, then install `webpack`, `webpack-cli` and `lodash`:
 
 ```bash
 npm init -y
-npm install --save-dev webpack lodash
```

Comment:
Add webpack-cli because when running `npx webpack` output 

```
CLI for webpack must be installed.
  webpack-cli (https://github.com/webpack/webpack-cli)

We will use "npm" to install the CLI via "npm install -D webpack-cli".
Do you want to install 'webpack-cli' (yes/no): no
You need to install 'webpack-cli' to use webpack via CLI.
```